hazelcast/src/test/java/com/hazelcast/quorum/cache/CacheQuorumListenerTest.java

```diff
@@ -240,12 +234,13 @@ public void onChange(QuorumEvent quorumEvent) {
         config.addQuorumConfig(quorumConfig);
         TestHazelcastInstanceFactory factory = createHazelcastInstanceFactory(3);
         HazelcastInstance instance1 = factory.newHazelcastInstance(config);
-        HazelcastInstance instance2 = factory.newHazelcastInstance(config);
+        factory.newHazelcastInstance(config);
         HazelcastServerCachingProvider cachingProvider = HazelcastServerCachingProvider.createCachingProvider(instance1);
         Cache<Object, Object> cache = cachingProvider.getCacheManager().getCache(cacheName);
         try {
             cache.put(generateKeyOwnedBy(instance1), 1);
         } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
I wasn't sure about this one, but you're right. I'll remove the try-catch and we'll see if it crashes.